src/scripts/client.js

```diff
@@ -42,10 +42,15 @@ const main = async () => {
 
   try {
     // Initialize the Voxeet SDK
-    // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
     // Please read the documentation at:
     // https://docs.dolby.io/communications-apis/docs/initializing-javascript
-    VoxeetSDK.initialize('customerKey', 'customerSecret');
+    // Grab your demos client token from your dashboard and insert into access_token
```

Comment:
Update to    // Generate a test client access token from the Dolby.io dashboard and insert into access_token variable